notebooks/max_grpc_msg_size.ipynb

```diff
@@ -20,6 +20,20 @@
     "```"
```

Comment:
<p>Nice one, may be worth contributing back this magic to Jupyter :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/2574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>